services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
```

Comment:
it's a good practice when your functions return smth

Reply:
I thought about it while implementing the functions, but the instructions didn't specify return value, so I assumed it to be `None`.